bigtop-packages/src/common/sqoop/patch3-hadoop333-denpendency-changes.diff

```diff
@@ -0,0 +1,40 @@
+diff --git a/ivy.xml b/ivy.xml
+index e4b45bfd..3b3d3b07 100644
+--- a/ivy.xml
++++ b/ivy.xml
+@@ -105,6 +105,10 @@ under the License.
+       conf="common->default;redist->default"/>
+     <dependency org="org.apache.commons" name="commons-lang3" rev="${commons-lang3.version}"
+       conf="common->default;redist->default"/>
++    <dependency org="mysql" name="mysql-connector-java" rev="${mysql-connector-java.version}"
```

Comment:
mysql-connector-java is licensed under GPL 2.0. We can not bundle this jar with our package. @kevinw66

Reply:
Removed @iwasakims